src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -83,6 +116,33 @@ public void setNumberOfChannels(String mediaInfoValue) {
 	}
 
 	/**
+	 * Get bits per sample for this audio track.
+	 * @return bits per sample (default is 16)
+	 */
+	public int getBitsperSample() {
+		return bitsperSample;
+	}
+
+	/**
+	 * Set bits per sample for this audio track.
+	 * @param bitsperSample bits per sample to set
+	 */
+	public void setBitsperSample(int bitsperSample) {
+		if (bitsperSample < 1) {
+			throw new IllegalArgumentException("Bits per sample can't be less than 1.");
+		}
```

Comment:
@Sami32 Yes, I imagine that can happen during "normal operation" and from plugins too, so the test is not the problem. That's why I said that you should accept 0 (or a negative value) in the setter, and the field should default to 0. But, in the getter you make an if statements that checks the value before returning it. If the value is < 1 you return the default (16?), if not you return the actual value.

That way, any value is accepted without throwing an exception, and the default will be used if the value isn't valid.